examples/nextjs-tiptap-novel/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@liveblocks-examples/nextjs-tiptap-novel",
+  "description": "This example shows how to build a collaborative text editor with Liveblocks, Novel, Tiptap, and Next.js.",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@liveblocks/client": "^2.12.0",
+    "@liveblocks/node": "^2.12.0",
+    "@liveblocks/react": "^2.12.0",
+    "@liveblocks/react-tiptap": "^2.12.0",
+    "@liveblocks/react-ui": "^2.12.0",
+    "@nanostores/react": "github:ai/react",
```

Comment:
🤔

Reply:
Changed it to the normal install, seems to be fine!